onnxruntime/python/tools/transformers/models/whisper/whisper_chain.py

```diff
@@ -42,8 +49,24 @@ def chain_model(args):
         "",  # attention mask
         "decoder_input_ids" if args.use_forced_decoder_ids else "",
         "logits_processor" if args.use_logits_processor else "",
+        "cross_qk_layer_head" if args.collect_cross_qk else "",
+        "extra_decoding_ids" if args.extra_decoding_ids else "",
     ]
+
     beam_outputs = ["sequences"]
+    if args.output_sequence_scores:
+        beam_outputs.append("sequence_scores")
```

Comment:
Can we use a similar setup to `beam_inputs` where the list of outputs shows the output order in a clear way?

```
beam_outputs = [
  "sequences",
  "sequence_scores" if args.output_sequence_scores else "",
  "scores" if args.output_scores else "",
  ...
]
```

Then the `while len(beam_outputs)` loops below can also be removed.

Reply:
I thinks this has the least impact to backward compatability.  also it looks not tidy as the form you suggested. Let's keep it.